lib/cadet_web/controllers/missions_controller.ex

```diff
@@ -0,0 +1,190 @@
+defmodule CadetWeb.MissionsController do
```

Comment:
Do these missions include paths, contests, and sidequests? In that case, can we rename this as Assessments?